plugins/modules/aci_fabric_node.py

```diff
@@ -59,6 +59,13 @@
     - I(remote_leaf_pool_id) is required if I(node_type) is C(remote).
     type: str
     aliases: [ pool_id ]
+  commission:
+    description:
+    - Commission adds a new node to the ACI fabric, which allows the node to participate in the fabric's operations and management.
+    - When commission is set to false, the node will only be decommissioned and not removed from the ACI fabric.
+    - This node can be recommissioned and does not need to be created as a new object.
```

Comment:
```suggestion
    - This option adds a new node to the ACI fabric, which allows the node to participate in the fabric's operations and management.
    - When set to false, the node will only be decommissioned and not removed from the ACI fabric.
    - Decommissioned nodes can be recommissioned and do not need to be created as new objects.
```
A slight re-wording suggestion.